functions.js

```diff
@@ -63,7 +63,74 @@ export function productHtmlBuilder(product) {
         //send back to the local storage
         setInLocalStorage('theCart', currentCart);
     });
+    return li;
+}
+
+
+//Bonus section
+//Delete from product list button, I put this on the specific button for each product, compared name because the rendered items dont have id associated, wish this was in original data model but decided to move forward on the bonus vs remake the model.
+function removeProduct(itemToDelete) {
```

Comment:
great work going above and beyond adding the remove bonus, definitely gives a smooth user experience in this way!